core/api/src/app/wallets/update-single-pending-invoice.ts

```diff
@@ -173,7 +183,7 @@ const processPendingInvoice = async ({
     currency: WalletCurrency.Btc,
   })
   if (receivedBtc instanceof Error) {
-    return ProcessPendingInvoiceResult.paidWithError(receivedBtc)
+    return ProcessPendingInvoiceResult.processAsPaidWithError(receivedBtc)
```

Comment:
if we have a conversion error it should be marked as paid/processed?

Reply:
True, this is the same kind of error as the `checkedToSats` one just above it, a validation error from an invalid amount passed. I made a change to treat them both the same and to decline the invoice if an invalid amount somehow comes through the process https://github.com/GaloyMoney/galoy/pull/4028/commits/cdcfcd764263b83477fb6f59b707d69ec4571f1e